### PR TITLE
Wow64 tweaks

### PR DESCRIPTION
**Pull request by Gcenx**
Upstreams wow64 system will use `wine64` for 32Bit binaries also, this is already being used within CrossOver-22.0.0

### CODE REVIEW

**Comment by Gcenx** [comment]
This is wrong, do we even still need to run regedit twice anymore?

**Comment by austin987** [comment]
AFAIK, yes. See https://learn.microsoft.com/en-us/windows/win32/sysinfo/32-bit-and-64-bit-application-data-in-the-registry

Unless something changed on the wine side that I missed (which is very possible, please let me know if I did).

**Comment by Gcenx** [comment]
I’ll do some additional testing with my CX22 based build and see how WoW64 handles this (it’s probably still correct for most things)

At least the paths I’d tired to use were incorrect as the standard regedit is within c:\windows but a 32Bit version in a 64Bit prefix would be within c:\windows\syswow64.

**Comment by austin987** [comment]
I can see how having explicit paths would be an improvement, so in that sense, I encourage this PR.

Remember that this needs to be tested in multiple configs. I.e., CX22 is a fine config to support, I'm more concerned with upstream wine/wine-staging, and also Linux/BSD, as well as OSX. That said, you're able to test OSX/CX22, which is also super helpful :). In any event, I appreciate the PR, and testing :).

In short though, this *must* pass 'make check' on linux/wine32 linux/wine64 (and ideally, osx/wine32 osx/wine64 bsd/wine32 bsd/wine64 cx/wine32 cx/wine64).

**Comment by austin987** [comment]
Also, after testing, if this tweak is still needed please reopen the issue :)

**Comment by Gcenx** [comment]
I’ll be testing against regular upstream along with WineCX21 (wine32on64) & WineCX22.

WineCX22 (wine-7.7) makes use of upstream still incomplete wow64 system meaning even 32Bit windows binaries will be launched via wine64, that’s the reason to have an exact PATH to the correct arch binary.

> Also, after testing, if this tweak is still needed please reopen the issue :)

Once I’m done with the expanded testing I’ll get back to this.
